firstvoices/pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
-addopts = --ds=firstvoices.settings_test --reuse-db
+addopts = --ds=firstvoices.settings_test --create-db
```

Comment:
In order to test async functions, we must re-create the db on test run to preserve fixtures.

Reply:
This solves migration conflicts during local development as well-- good change